workspaces/config/lib/index.js

```diff
@@ -695,6 +687,7 @@ class Config {
       }
 
       if (this.localPrefix && hasPackageJson) {
+        const rpj = require('read-package-json-fast')
```

Comment:
We use none of the normalization here, we should consider using the underlying JSON parser in this situation.